webapp/src/DOMJudgeBundle/Resources/views/jury/config.html.twig

```diff
@@ -0,0 +1,62 @@
+{% extends "@DOMJudge/jury/base.html.twig" %}
+{% import _self as self %}
+{% import "@DOMJudge/jury/jury_macros.twig" as macros %}
+
+{% block title %}Configuration - {{ parent() }}{% endblock %}
+
+{% block extrahead %}
+    {{ parent() }}
+    {{ macros.table_extrahead() }}
```

Comment:
I am guessing we don't use datatables here. In that case the whole block and macro import can be removed.

Reply:
We need it, that's how I import the eyecandy toggles.